sdk/template/azure-template/CHANGELOG.md

```diff
@@ -2,103 +2,103 @@
 
 ## 1.0.0 (2021-06-03)
 
-* Validate template project ahead of release
+- Validate template project ahead of release
```

Comment:
I would move this to another PR, so it's not smuggled within attestation release.
Thank you for changing it for template.